rust_team_data/src/v1.rs

```diff
@@ -1,8 +1,15 @@
+use std::collections::BTreeSet;
+
 use indexmap::IndexMap;
 use serde::{Deserialize, Serialize};
 
 pub static BASE_URL: &str = "https://team-api.infra.rust-lang.org/v1";
 
+#[derive(Debug, Clone, Serialize, Deserialize, PartialEq, Default)]
```

Comment:
I don't think we need to expose this to the JSON API? It's an internal implementation detail. Can't you just pass `&Config` from `team` to `sync-team` in `run_sync_team`?